.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+- '8'
+services:
+  - docker
+  - rabbitmq
+  - redis
+script: npm test:ci
+notifications:
+  slack:
+    secure: J97h63QLRfK2DV41dYv42T0b6mRBxe3joDQP2EeEjpyIYQ52i2so6m5NA1GVJydzQ1uEYcxicX6GVeox0lt3PSqPAMYr9Tb/7pJ9mBOAAEoSpkgRcNsMItbEkJGGL5IfvTEy9/2JiXhrNbrCGlJepO72zq/Hjkc5oIYs7PTNCAhXk3Ys0HyQKnHxvwq3DAk2t39PaLAoNDbZuqZ95SqbvZJjehjv6fXolUkuwT090rThyCK43ATaDaXKWFEIVsiYxnsiu7TLqvIG1b2mLa2Zo0giQrH+5DOrT8a73ZJHdPzxI1OknC0Ezy5GdUsXaIV4Bqhdb2CMGexY+XieBn/hMgWFD+yic5vVvnXh0Z3QBIROR4T2PF6ubZfick9Yx7/z+eo/HEIZW7QpKN+4y0hBaK2KQr6WyXCMg8xoTEFgdakqgYtZdc/egTlHb1KrSYxviNkgidVIiApbTzhnVRwB/Pj727m9g2ffn73nS7GDzMUxUvIE6L8tk238JEmx77XmkFAKjdAeTAf/eCm9J6nZH8kJ8I1EMS08gUymacBb5fbUb8mRSmNPnTKCgMjwkKPaKMDyqqZcdu7sYv/lfvH4/jtuc0pXSIeOoi3hoNFmbazf3xnigS4Vn0rDw1UpZ65ZLSCCP1mtbfN1fXekhtuOfxeJXwn49SZnyt6vwOhiQDA=
```

Comment:
Did you copy this from one of the other repos? I'm not sure if this token will work. (If it does, the slack integration for travis is totally insecure).